app/upload.go

```diff
@@ -19,6 +19,65 @@ import (
 
 const minFirstPartSize = 5 * 1024 * 1024 // 5MB
 
+func (a *App) runPluginsHook(info *model.FileInfo, file io.Reader) *model.AppError {
+	// call plugins upload hook
+	if pluginsEnvironment := a.GetPluginsEnvironment(); pluginsEnvironment != nil {
```

Comment:
Nit: we can return early if `pluginsEnvironment == nil` to save a one level indentation.

Reply:
Yeah, nice catch. This was moved from inside another function so I missed the possible improvement. Will update :+1: